### PR TITLE
Upgrade Ruby to 2.7

### PR DESCRIPTION
**Pull request by rianniello**

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ phases:
 
       - "[ -e .ruby-version ] && rbenv install $(cat .ruby-version) --skip-existing"
 
-      - gem install bundler
+      - gem install bundler -v 1.17.2
```
Comment (rcaught) [inline buildspec.yml]: As talked about, this is ok for now.  Maybe add the reason for pinning as a comment here?

Comment (rianniello) [reply buildspec.yml]: https://github.com/ausaccessfed/validator-service/pull/182/commits/3d68faf97f7bf53e1dffe3bc6c856188a15b45f1

Comment (rianniello) [reply buildspec.yml]: https://github.com/ausaccessfed/validator-service/pull/182/commits/02d9e3d22647ded81128e264ac66f586f4e93ef7

**Comment by vladimir-mencl-eresearch** [comment]
Hi,

FYI, when I've pushed this through our deployment flow (Ansible), I got the following in the `bundle exec rails assets:precompile` output:
```
/opt/rbenv/attrval/2.7.0/gems/bundler-1.17.2/lib/bundler/rubygems_integration.rb:200: warning: constant Gem::ConfigMap is deprecated
/opt/rbenv/attrval/2.7.0/gems/bundler-1.17.2/lib/bundler/rubygems_integration.rb:200: warning: constant Gem::ConfigMap is deprecated
/opt/rbenv/attrval/2.7.0/ruby/2.7.0/gems/will_paginate-3.2.1/lib/will_paginate/railtie.rb:66: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/opt/rbenv/attrval/2.7.0/ruby/2.7.0/gems/activesupport-5.2.4.1/lib/active_support/rescuable.rb:51: warning: The called method `rescue_from' is defined here
```

Still works all fine, just produced this output.  Probably outcome of a new Ruby version but older version of some Gems .... and hopefully harmless.

**Comment by smangelsdorf** [comment]
Thanks Vlad. Yes, it's a deprecation that won't be a hard error until Ruby 3.x, and I suppose it will take some time for gems to update their syntax. Hopefully we'll be rid of all deprecations next time we do a dependency update.

**Pull request merged**